decrypt_test.go

```diff
@@ -133,7 +133,7 @@ yoP7LBNaX/r83/SsGGbtW618sar+27wWRlGeZvFq8hwh+qyj3S7wtY7wx5weR7eZ
 S9K/MB9npTbIjDUidByTIhpk0hS55s1jF6U0wlmV0oUmC4wBIebyo1cL/CMPRv+a
 z/D9/ZvUoaotiATRWSKLCmxapcmGbMxhHHYWYWtzPCt1
 -----END CERTIFICATE-----
------BEGIN RSA PRIVATE KEY-----
+-----BEGIN PRIVATE KEY-----
```

Comment:
According to the de-facto standards, this should be `RSA PRIVATE KEY` as it is encoded using PKCS #1.

```console
$ openssl genrsa
-----BEGIN RSA PRIVATE KEY-----
MIIEpAIBAAKCAQEAvWoH5dogD9w+pCVkuxY6TWad7LVcSTPV2UnLMgz4gbit9uPx
...
-----END RSA PRIVATE KEY-----
$ openssl genrsa | openssl pkcs8 -topk8 -nocrypt
-----BEGIN PRIVATE KEY-----
MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQC4Ih+uBI2U3uq3
...
-----END PRIVATE KEY-----
```